x-pack/test/cases_api_integration/security_and_spaces/tests/common/internal/bulk_create_attachments.ts

```diff
@@ -1343,10 +1355,24 @@ export default ({ getService }: FtrProviderContext): void => {
         const attachments = await getAllComments({ supertest, caseId: postedCase.id });
         expect(attachments.length).to.eql(2);
 
-        const secondAttachment = attachments[1] as CommentRequestAlertType;
+        let foundAlert3 = false;
```

Comment:
Is this connected to

> There were a few places we were only waiting for 1 alert to be created but we were expecting 2 alerts to be there which resulted in undefined when accessing the second alert and produced the _can't find id

?

But if we had `expect(attachments.length).to.eql(2);`(on line 1356) the test would fail before we tried to access the second alert.